packages/xrpl/src/models/transactions/common.ts

```diff
@@ -9,15 +9,15 @@ import {
   AuthorizeCredential,
   Currency,
   IssuedCurrencyAmount,
+  MPTAmount,
   Memo,
   Signer,
   XChainBridge,
-  MPTAmount,
 } from '../common'
 import { onlyHasFields } from '../utils'
 
 const MEMO_SIZE = 3
-const MAX_CREDENTIALS_LIST_LENGTH = 8
+export const MAX_CREDENTIALS_LIST_LENGTH = 8
```

Comment:
It's better to use JS terminology for array than using Python's list.
```suggestion
export const MAX_CREDENTIALS_ARRAY_LENGTH = 8
```

Reply:
[2080c21](https://github.com/XRPLF/xrpl.js/pull/2874/commits/2080c21bd425a596f844d2898ed65c465f19adfd)